content/applications/websites/website/configuration/cookies_bar.rst

```diff
@@ -0,0 +1,71 @@
+===========
+Cookies bar
+===========
+
+Cookies are small text files sent to your device when you visit a website. They are processed and
+stored by your browser and contain information about your visit, such as login data, location,
+language, etc. There are two main types of cookies:
+
+- Essential cookies, which are necessary for the website to function properly;
+- Non-essential or optional cookies, which are used to analyze your behavior or display
+  advertisements.
+
+Notifying users about data collection, as well as its methods and purposes, is required by data
+protection laws such as `GDPR <https://gdpr.eu>`_. Cookies bars are commonly used to fulfill this
+obligation in a user-friendly and transparent manner. They are displayed immediately upon a user's
+first visit to inform them that the website uses cookies and let them decide whether they want to
+store non-essential cookies on their device.
+
+Configuration
+=============
+
+To enable the cookies bar on your Odoo website, go to :menuselection:`Website > Configuration` and
+enable :guilabel:`Cookies bar` in the :guilabel:`Privacy` section.
+
+ .. note::
+    If you have :doc:`multiple websites <../configuration/multi_website>`, don't forget to first
+    select the website for which you want to activate the cookies bar in the :guilabel:`Settings of
+    Website` field at the top of the page.
+
```

Comment:
Add a small sentence to explain you can customize the bar/popup and edit content of the privacy page (with custom anchors to the titles)